pom.xml

```diff
@@ -85,9 +85,10 @@
         <kafkaVersion>0.10.0.1</kafkaVersion>
 
         <!-- Define which Curator version -->
-        <curatorVersion>2.12.0</curatorVersion>
-
+        <curatorVersion>4.0.0</curatorVersion>
 
+        <!-- Define which Curator Test version -->
+        <curatorTestVersion>2.12.0</curatorTestVersion>
```

Comment:
This is the internal zk test server that we use in tests.  From digging around on the curator website and in JIRA tickets, as far as I can tell this is the latest release that actually works, even though there are releases published up through 4.0.0.

Reply:
What's the scoop with this in 4.0.0? This seems really weird...

Reply:
No idea.  It bombs if you move it up to any version past 2.12.0.  I'll try to dig in more tomorrow and see if I can find out more.